test/new-e2e/tests/agent-metric-logs/log-agent/utils/file_tailing_utils.go

```diff
@@ -42,14 +59,14 @@ func appendLog(s *LinuxFakeintakeSuite, content string, recurrence int) {
 		checkCmd = fmt.Sprintf("sudo cat %s", logPath)
 	}
 
-	s.EventuallyWithT(func(c *assert.CollectT) {
+	th.EventuallyWithT(func(c *assert.CollectT) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You can use `assert.EventuallyWithT` instead, no need to pass the function through the interface
```suggestion
	assert.EventuallyWithT(t, func(c *assert.CollectT) {
```